clients/web/src/rest.js

```diff
@@ -171,6 +172,10 @@ let restRequest = function (opts) {
         throw new Error('restRequest requires a "url" argument');
     }
     args.url = `${getApiRoot()}${args.url.substring(0, 1) === '/' ? '' : '/'}${args.url}`;
+    if (args.type) {
+        args.method = args.type;
+        delete args.method;
```

Comment:
Might be worth a short comment here, either briefly explaining this block or just linking to this PR